src/JsonSchema.js

```diff
@@ -56,15 +81,16 @@ class JsonSchema {
             return referencedSchema.getFieldValue(fieldName, mergeFunction, mappingFunction);
         }
         const rawValue = this.schema[fieldName];
-        const value = mappingFunction ? mappingFunction(rawValue, this.scope) : rawValue;
-        if (this.schema.allOf) {
+        let value = mappingFunction ? mappingFunction(rawValue, this.parserConfig, this.scope) : rawValue;
```

Comment:
so far the only non-backwards-compatible change in comparison to `v1.2.0`: the `parserConfig` being passed on as second parameter instead of the `scope`.

Since there will probably be more structural changes required in order to show the `anyOf`/`oneOf` options as alternatives, there is no need to try to make this backwards-compatible here (assumption being that this is not used so far by anyone).